dashboard/test/ui/features/instructions/feedback_tab_eyes.feature

```diff
@@ -58,8 +58,9 @@ Feature: Feedback Tab Visibility
     Then I sign out
     And I sign in as "Lillian"
     And I am on "http://studio.code.org/s/allthethings/stage/38/puzzle/1?enableExperiments=2019-mini-rubric"
-    And I press the first ".uitest-feedback" element
-    And I wait for the page to fully load
+    And I wait to see ".uitest-feedback"
```

Comment:
I think this line has you covered here, but for next time one good practice is to do `And I wait for the page to fully load` to make sure the page is at least a certain amount loaded after each time you navigate to a script level page (only works on pages with a runButton)